### PR TITLE
feat(orderUpdated): Add support for delivery methods

### LINKED ISSUE #14: Support delivery info
We would like to support the update actions "add deliveries".
(see https://dev.commercetools.com/http-api-projects-orders.html#add-delivery)

Due to the complexity of the delivery object, we are not able to structure an easy-to-use csv format. We have suggested a new structure for this (see possible solution).

_GroupByDelivery, ItemID, quantity and parcel.id_ are mandatory fields.

### Current Behavior
Not supported currently

### Possible Solution
There could be a GroupByDeliveryID which is only there to separate the deliveries from each other. So all line items with the same GroupByDeliveryID belongs to the same Delivery.
Inside this, the parcelID is available to separate the parcels from each other.
Clarify: Dies the ID have to be unique over all parcels or just inside the delivery?

### PR DESCRIPTION
**Pull request by junajan**
#### Summary
This PR resolves #14 

#### Description
It adds a support for actions `addDelivery` and `addParcelToDelivery`

#### Reviewers

- [ ] Reviewer 1 has approved the PR
- [ ] Reviewer 2 has approved the PR

#### TODO

- Tests
    - [x] Unit
    - [x] Integration
    - [ ] Acceptance
- [ ] Documentation

### CODE REVIEW

**Review by adnasa — APPROVED** [review]
This looks to me codewise 👍

File: src/build-order-actions.js [hunk 1]
```diff
@@ -109,6 +110,38 @@ const buildOrderMethods = {
...
+
+  shippingInfo: (order, existingOrder) => {
+    const sync = new OrderSync()
+    const enabledActions = [
+      'addDelivery',
+      'addParcelToDelivery',
+    ]
+
+    /* eslint-disable no-param-reassign */
+    order = _.defaults(order, {
```
Comment (wizzy25) [inline src/build-order-actions.js]: Why are you reassigning the parameter? You could have used a new variable entirely.

```js
    const modOrder = { shippingInfo: { deliveries: [] }, ...order }
    const modExistOrder = { shippingInfo: { deliveries: [] }, ...existingOrder }
```
And then you wouldn't need underscore here ;)

**Review by wizzy25 — CHANGES_REQUESTED** [review]
Looks good... Just a few things I noticed

**Comment by hisabimbola** [comment]
Let's keep this on hold till when it's implemented in the `csv-parser-order` module...

**Comment by junajan** [comment]
There was added this PR https://github.com/commercetools/csv-parser-orders/pull/18 which adds a possibility to parse deliveries CSV into JSON object. This JSON object can be then taken by this module and imported into API.

File: src/build-order-actions.js (hunk 1 above)
Comment (junajan) [reply src/build-order-actions.js]: Simply beautiful .. thanks, updated here f84a39e

**Comment by junajan** [comment]
This PR is currently blocked by this issue on the SDK https://github.com/sphereio/sphere-node-sdk/issues/229

**Review by wizzy25 — APPROVED** [review]

**Review by hisabimbola — APPROVED** [review]

**Pull request merged**